### DISCUSSION TITLE
Why are you using a Mac and will you ever go back to Linux?

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#23] [Q&A] [answered]
Why are you using a mac?

**Comment by tallguyjenks** [comment, marked as answer]
So I get asked this a lot, why am I using a Mac? Why not "x, y, z...?" 

Firstly I'll list these preferences here: 
- I like *nix based systems
    - the CLI
    - the directory structure 
    - the way you can script on the system with shell script and cron
    - and much more
- I've used the big 3 (Windows, Mac, and Linux) of them all my favorite is Linux, by far.

**"So why are you on Mac then?!?!"**

A little back story: I started my [YouTube Channel](https://www.youtube.com/c/BryanJenksTech?sub_confirmation=1) making videos on Linux things while running Arch Linux on a Thinkpad x220 that had an old i7, 16GB of RAM though its technically not able to handle that much, and the thing couldn't even rend a video with jump cuts in OpenShot without overheating and crashing. So I basically couldn't edit videos unless I did it on the command line with FFmpeg which I did a little of until I quickly tired of it. Then I also happened to start school again for a bachelor's degree program and the supported operating systems were macOS and Windows. Absolutely no flavors of Linux as Linux doesn't support the Proctoring Malware that remote universities demand that you install. 

So it was either macOS or Windows. Now with that choice, I went with macOS for a few reasons:

1. I've never used macOS and the geek in me wanted to try it
2. I was already in the ecosystem with an iPad, iPhone, and with the MacBook pro purchase, I also got AirPods for free. 
3. macOS is a *nix based system so I could use the CLI much the same way
4. I have a mac system to test scripts on too now 
5. The model and specs I bought mean that this thing is a tank and can handle all the video editing, streaming, and various needs I had for it with ease. and for gaming, I can use Bootcamp and have a windows partition on the machine solely for gaming when desired. 
6. Resale value

So here's the breakdown of my responses, questions in bold, answers as sub-bullets

- **Why not use windows and just dual boot Linux, or just use WSL?**
    - I dislike WSL, customizing the system is hard with WSL and I was not a fan and I didn't want to dual boot because I just didn't want to use windows as my daily driver anymore at all.
- **Do you miss Linux?**
    - YES. I love Linux and I'm not getting too invested in the ecosystem to fear leaving. Not going to lie it IS nice and comfy how everything is seamless and works together, but that raw and scrappy hacked together system with some shell scripts and terminal commands in Linux really feels like you're the one with all the power and no spookiness from apple will mess with you.
- **Will you ever go back to Linux?**
    - Yes, I do intend to build a custom desktop running Arch as my daily driver and also have Arch on some Thinkpads because I truly love them. I like Arch and how much it teaches you just by setting it up and maintaining it. So ultimately I do intend to return to Linux after school and when I have time to configure a system from scratch which at the current time I do not.

**Reply by PaulFWatts** [reply]
I think you made a good choice.

The reliability (hardware and software), ease of upgrades, desktop and mobile integrated ecosystem, and the power of the ALMOST linux, BSD,  underneath the GUI are compelling reasons to go Mac in my humble opinion 👍 .

Is only going to get even better with the M1 and it's desktop successors in the near future.

**Reply by tallguyjenks** [reply]
I However do not like how locked down the Mac's are and will ultimately return to Arch Linux as the control and customization of that system is very much to my liking. There is no denying the power of the Apple ecosystem though
